<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>GraphBuilder</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="GraphView" name="graphView" native="true">
                        <property name="minimumSize">
                            <size>
                                <width>500</width>
                                <height>500</height>
                            </size>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="CommandList" name="commandList">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>0</height>
                                    </size>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Search for blocks</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="blockEdit">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>256</width>
                                        <height>0</height>
                                    </size>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>37</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionOpen"/>
                <addaction name="actionSave"/>
                <addaction name="actionSaveAs"/>
            </widget>
            <addaction name="menuFile"/>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionOpen">
            <property name="text">
                <string>Open</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionSave">
            <property name="text">
                <string>Save</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+S</string>
            </property>
        </action>
        <action name="actionSaveAs">
            <property name="text">
                <string>Save As</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Shift+S</string>
            </property>
        </action>
    </widget>
    <customwidgets>
        <customwidget>
            <class>GraphView</class>
            <extends>QWidget</extends>
            <header>graphview.h</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>CommandList</class>
            <extends>QListWidget</extends>
            <header>commandlist.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
